src/manage.c

```diff
@@ -177,7 +178,7 @@ u8 best_prn(void)
       if (acq_prn_param[prn].state == ACQ_PRN_TRIED)
         acq_prn_param[prn].state = ACQ_PRN_UNTRIED;
     }
-    printf("acq: restarting PRN search\n");
+    log_info("acq: restarting PRN search\n");
```

Comment:
It may be useful to push such a log tag up as a separate string, so the console or other log inspection tool could filter by tag/subsystem.